lib/fork.js

```diff
@@ -153,7 +153,14 @@ module.exports = function (file, opts, execArgv) {
 		return promise;
 	};
 
+	var isExitInitiated = false;
```

Comment:
What change made this necessary? Maybe worth using [`onetime`](https://github.com/sindresorhus/onetime) here?